test_conformance/commonfns/test_base.h

```diff
@@ -0,0 +1,194 @@
+//
+// Copyright (c) 2022 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef _TEST_COMMONFNS_BASE_H
```

Comment:
This symbol is reserved:

> All names which begin with two underscores, **or an underscore and a capital letter**, are reserved for the compiler and library

So I'd suggest

```suggestion
#ifndef TEST_COMMONFNS_BASE_H
```
(and propagate to the `#define` and `#endif`)

Reply:
It's done